packages/react-components/src/organisms/SharedResearchOutputButtons.tsx

```diff
@@ -121,7 +127,22 @@ const SharedResearchOutputButtons: React.FC<
           </Button>
         </div>
       )}
-      {canVersionResearchOutput && published && (
+      {isManuscriptOutputFlagEnabled &&
+        canVersionResearchOutput &&
+        published &&
+        hasRelatedManuscript && (
+          <div css={leftButtons}>
+            <Button
+              noMargin
+              small
+              primary
+              onClick={checkForNewerManuscriptVersion}
+            >
+              <VersionIcon /> Import Manuscript Version
+            </Button>
+          </div>
+        )}
+      {canVersionResearchOutput && published && !hasRelatedManuscript && (
```

Comment:
maybe we can add `!isManuscriptOutputFlagEnabled` here? because otherwise the `Add version` button just disappears when the flag is not enabled